exec/csnex/Cell.cs

```diff
@@ -165,16 +181,20 @@ public enum Type
         }
 
         public Type type { get; private set; }
+
+        public UInt64 AllocNum { get; private set; }
```

Comment:
Why does this declare a setter? Can you just say `{ get; }` because that's all you need?

Reply:
Ah.  Good call.  I assumed I needed one for the constructor to be able to set it, forgetting that the constructor isn't accessing it as a property, but rather a local variable.  Removed the `set;`.